WordPressAuthenticator/Extensions/UIButton+Styles.swift

```diff
@@ -0,0 +1,9 @@
+import UIKit
+
+extension UIButton {
+    /// Applies default styles to a plain text button.
+    func applyTextButtonStyle() {
```

Comment:
I think this method might be better suited added to WPStyleGuide instead of UIButton. You could add it to `WPStyleGuide+Login.swift`, which is where we have methods to configure other buttons like the 1Password button (you may also want to rename it to `configureTextButton` instead of `applyStyle` as that might be more consistent).